qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/AttachmentsPage.java

```diff
@@ -34,6 +41,22 @@ public void setFileUrl(String fileUrl) {
 		WebElement element = driver.findElement(By.xpath("//input[@type='file']"));
 		element.sendKeys(fileUrl);
 	}
+	
+	public String createRandomFile() throws IOException {
+		File tempFile = File.createTempFile("file", ".pdf");
+		tempFile.deleteOnExit();
+		try (InputStream pdf = getClass().getClassLoader().getResourceAsStream("data/file.pdf")) {
+			try (Writer writer = new FileWriter(tempFile)) {
+				IOUtils.copy(pdf, writer, StandardCharsets.UTF_8);
+			}
+		}
+		return tempFile.getAbsolutePath();
+	}
+	
+	public void getRandomFile() throws IOException {
```

Comment:
I would probably name this method something like `attachFile()` as I think that's a little clearer what it does. Definitely `get` is not the right verb here though (there's no result returned).

Reply:
@ibacher , I will rename the method to attachFile().